src/seedsigner/views/seed_views.py

```diff
@@ -367,44 +367,49 @@ def run(self):
         self.seed.set_passphrase(ret_dict["passphrase"])
 
         if "is_back_button" in ret_dict:
```

Comment:
All of the if/else logic can now be simplified to:

```suggestion
        if "is_back_button" in ret_dict or len(self.seed.passphrase) == 0:
            return Destination(SeedAddPassphraseExitDialogView)
        
        else:
            return Destination(SeedReviewPassphraseView)
```

(the `else` isn't even necessary but I tend to prefer seeing the explicit if/else balance rather than having a "naked" default return value)